CMakeLists.txt

```
@@ -29,7 +29,7 @@ endif()
 
 project(microxrcedds VERSION "2.0.0")
 
-set(_client_tag develop)
+set(_client_tag feature/add_history_depth)
```

When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/266 merged:
```suggestion
set(_client_tag develop)
```